thirdparty/misc/ok_color.h

```diff
@@ -22,9 +22,8 @@
 #include <cmath>
 #include <cfloat>
 
-class ok_color
+namespace ok_color
```

Comment:
It seems like the upstream `ok_color.h` has already done this change: https://github.com/bottosson/bottosson.github.io/blob/master/misc/ok_color.h

Instead of making our own changes, we may wish to just update our copy of the library.

Reply:
It's not that upstream made this change, it's that upstream started as namespace and stayed that way.
Our copy of ok_color.h was changed for some reason, and I'm simply changing it back.

Reply:
The upstream file uses pragma once instead of a header guard.
The current version in this PR is up to date with upstream, with minor adjustments to make it work in Godot.